libraries/standard/mqtt/include/mqtt.h

```diff
@@ -347,6 +498,40 @@ MQTTStatus_t MQTT_Ping( MQTTContext_t * pContext );
  * #MQTTBadParameter if invalid parameters are passed;
  * #MQTTSendFailed if transport write failed;
  * #MQTTSuccess otherwise.
+ * 
+ * <b>Example</b>
+ * @code{c}
+ *
+ * //Variables used in this example
+ * MQTTStatus_t status;
+ * MQTTSubscribeInfo_t unsubscribeList[ NUMBER_OF_SUBSCRIPTIONS ] = { 0 };
+ * uint16_t packetId;
+ * //This is assumed to have been initialized and connected before calling this function.
+ * MQTTContext_t * pContext;
+ * //This is assumed to be a list of filters we want to unsubscribe from.
+ * const char * filter[ NUMBER_OF_SUBSCRIPTIONS ];
+ * 
+ * //Set information for each unsubscription request
+ * for( int i = 0; i < NUMBER_OF_SUBSCRIPTIONS; i++ )
+ * {
+ *      //Each subscription needs a topic filter, 
```

Comment:
unsubscription

Reply:
wait is this a word? lol

Reply:
Maybe we can just use unsubscribe?

Reply:
Thanks, it's not. I changed each instance to "unsubscribe request" or "unsubscribing"